README.md

```diff
@@ -1,2 +1,457 @@
-# unione-php
-UniOne Email API PHP Library
+# UniOne PHP client
+
+[![Latest Stable Version](http://poser.pugx.org/unione/unione-php/v)](https://packagist.org/packages/unione/unione-php)
+[![Total Downloads](http://poser.pugx.org/unione/unione-php/downloads)](https://packagist.org/packages/unione/unione-php)
+
+This is the UniOne PHP SDK. This SDK contains methods for easily interacting
+with the UniOne API. For additional
+examples, please see our official documentation at https://docs.unione.io/en/web-api-ref#web-api
+
+# Installation
+
+To install the SDK, you will need to be using [Composer](http://getcomposer.org/)
+in your project.
+If you aren't using Composer yet, it's really simple! Here's how to install
+composer:
+
+```bash
+curl -sS https://getcomposer.org/installer | php
+```
+
+Use Composer to install the package:
+
+```bash
+composer require unione/unione-php
+```
+
+## Usage
```

Comment:
Why this is heading of 2nd level, which means it's below "Installation" section?